include/dsVideoPort.h

```diff
@@ -941,7 +949,8 @@ dsError_t dsGetHdmiPreference(intptr_t handle, dsHdcpProtocolVersion_t *hdcpCurr
 /**
  * @brief Gets the IgnoreEDID status variable set in the device.
  *
- * This function is used to retrieve the status variable in order to determine whether to ignore the EDID data or not.
+ * For source devices, this function is used to retrieve the status variable in order to determine whether to ignore the EDID data or not.
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
```

Comment:
Do we need this API? Is there A STB implementation

Reply:
XiOne has HAL Implementation for Ignore EDID status